cioos_data_transform/setup.py

```diff
@@ -13,6 +13,6 @@
     long_description_content_type="text/markdown",
     url="https://github.com/cioos-siooc/cioos-siooc_data_transform",
     packages=setuptools.find_packages(),
-    install_requires=["numpy", "fortranformat", "netCDF4", "pytz", "shapely", "gsw==3.3.1"],
+    install_requires=["numpy", "fortranformat", "netCDF4", "pytz", "shapely", "gsw==3.3.1", "isodate"],
```

Comment:
can't you use pytz and/or datetime module instead of isodate ?

Reply:
Doesn't seem like pytz or datetime can convert timedelta values to a iso duration format but pandas can do! Just made some changes.

Reply:
great ! we can remove the isodate dependency then ?